lnbits/core/migrations.py

```diff
@@ -188,3 +192,64 @@ async def m005_balance_check_balance_notify(db):
         );
     """
     )
+
+
+async def m006_add_invoice_expiry_to_apipayments(db):
+    """
+    Adds invoice expiry field to apipayments and precomputes them for
+    existing entries
+    """
+    try:
+        await db.execute("ALTER TABLE apipayments ADD COLUMN expiry TIMESTAMP")
+    except OperationalError:
+        pass
```

Comment:
- should `raise` instead of `pass`, migration should not continue if `expiry` column was not added

Reply:
- maybe `ALTER TABLE apipayments` and `UPDATE apipayments` should be two separate migrations (`m006_` and `m007`)

- `ALTER TABLE apipayments` migration can do a check if `expiry` column is present and skip executing the command if true

Reply:
Separated migrations to `m006` and `m007`.